sematic/logging.py

```diff
@@ -0,0 +1,57 @@
+# Standard Library
+import os
+
+# Sematic
+from sematic.config.config import get_config
+
+
+def make_log_config(log_to_disk: bool = False):
```

Comment:
So that log formatting can be more consistent, in particular that it will match between the normal server logs and the logs produced by the cleaner.